dashboards/harbor/harbor.json

```diff
@@ -0,0 +1,221 @@
+{
+  "name": "Harbor Prometheus",
+  "description": null,
+  "permissions": "PRIVATE",
```

Comment:
The `permissions` field is not a valid field as part of dashboards that are installed in Quickstarts. See the [relevant docs](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards).

```suggestion
```